LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright (c) 2024 SHIZZLE
```

Comment:
SHIZZLE is unprofessional, how about "Highdashers Developer Group"?

Reply:
Changed, check again :)